### DISCUSSION TITLE
关于使用emby4.7.x用户如何继续使用作者大大的插件

### DISCUSSION THREAD

**Discussion by z3shan33** [metatube-community/jellyfin-plugin-metatube#292] [General]
拉作者b8a68d8的这次提交的代码（就是发布4.8.0版本支持的上一次提交）。找到ScheduledTasks目录下的UpdatePluginTask注释掉这个类，这样就不会自动更新插件了，然后打包，将打包后的dll覆盖到插件目录。

**Comment by pentium2023** [comment]
小白搞不来啊，希望有大佬封装个不会自动更新的METATUBE.DLL，我群晖上的emby4.7.14开心版，插件总是自动更新到最新，然后失效...

**Comment by z3shan33** [comment]
[Emby.MetaTube@v2024.218.833.0.zip](https://github.com/metatube-community/jellyfin-plugin-metatube/files/14382943/Emby.MetaTube%40v2024.218.833.0.zip)

**Reply by pentium2023** [reply]
太感谢了，10086个赞给大佬~~
